benchmarks/benchmark_serving.py

```diff
@@ -0,0 +1,256 @@
+# Adapted from https://github.com/vllm-project/vllm/blob/main/benchmarks/benchmark_serving.py
```

Comment:
Can we use the same format like files we implemented the openAI API? Adding the apache license header.